apps/cli-daemon/src/app/utils/extract -value-object.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { extractValueObject } from './extract -value-object';
```

Comment:
I think there is an accidental space in the file name

Reply:
fixed :)